assets/scripts/segments/components.json

```diff
@@ -917,18 +917,25 @@
       }
     },
     "food-truck": {
+      "baz": 3,
```

Comment:
Is this going to be expanded to anything else in the future?

Reply:
(Or, is this doing anything right now?)

Reply:
✂️